pagopa/io-app
# chore: [IOBP-690] Add pre-selection psp logic for payment methods with pspId indicated

## Short description
This PR adds automatic pre-selected PSP selection logic in the past (in this case useful for PayPal fast payment method). If the get user wallets returns us a wallet with an indicated pspId, it means that in the calculateFees if that method is selected, only that id needs to be sent to the `idPspList`.

## List of changes proposed in this pull request
- Added a logic to select the pspId from the user wallet if present and send it to the calculateFees request action

## How to test
- Checkout this dev-server PR: https://github.com/pagopa/io-dev-api-server/pull/386
- If you select "Pagamento veloce con PayPal" from the payment method selection list, you should be able to navigate directly to the payment confirm screen.

## Preview

https://github.com/pagopa/io-app/assets/34343582/feb88ba2-2559-4888-8d86-660eb1800882
